test/12-device-config.spec.ts

```diff
@@ -642,137 +804,124 @@ describe('Device Backend Config', () => {
 			readFileStub.restore();
 		});
 	});
-});
 
-describe('getRequiredSteps', () => {
-	const bootMountPoint = path.join(
-		constants.rootMountPoint,
-		constants.bootMountPoint,
-	);
-	const configJson = 'test/data/config.json';
-	const configTxt = path.join(bootMountPoint, 'config.txt');
-	const deviceTypeJson = path.join(bootMountPoint, 'device-type.json');
-	const osRelease = path.join(constants.rootMountPoint, '/etc/os-release');
-	const splash = path.join(bootMountPoint, 'splash/balena-logo.png');
-
-	// TODO: something like this could be done as a fixture instead of
-	// doing the file initialisation on 00-init.ts
-	const mockFs = () => {
-		mock({
-			// This is only needed so config.get doesn't fail
-			[configJson]: JSON.stringify({}),
-			[configTxt]: stripIndent`
+	describe('getRequiredSteps', () => {
+		const splash = path.join(bootMountPoint, 'splash/balena-logo.png');
+
+		// TODO: something like this could be done as a fixture instead of
+		// doing the file initialisation on 00-init.ts
+		const mockFs = () => {
+			mock({
+				// This is only needed so config.get doesn't fail
+				[configJson]: JSON.stringify({}),
+				[configTxt]: stripIndent`
 				enable_uart=true
 			`,
-			[osRelease]: stripIndent`
+				[osRelease]: stripIndent`
```

Comment:
Anything off about indents here?

Reply:
I don't know why the indents got messed up here, they looked ok in my editor.